test/acceptance-tests/workspace-devtime/step-definitions/reading-artifacts/findModelProperty.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+var util = require('util');
+var async = require('async');
+var app = require('../../../../../');
+var loopback = require('loopback');
+var chai = require('chai');
```

Comment:
helpers would be added in the next iteration. this iteration is only to add basic test cases